src/shared/prerendered-app/colors.css

```diff
@@ -0,0 +1,19 @@
+html {
+  --pink: #ff3385;
+  --hot-pink: #ff0066;
+  --white: #fff;
+  --dim-blue: #0a7bcc;
+  --deep-blue: #09f;
+  --light-blue: #76c8ff;
+  --light-gray: #eaeaea;
+  --dark-text: #343a3e;
+
+  /* Old stuff: */
```

Comment:
I intend to remove/refactor these as I apply the redesign to the rest of the app